README.md

```diff
@@ -2,77 +2,55 @@
+`cd` into hackton-frontend and run `npm start` or `yarn start` in the console after cloning the repository.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+### Prerequisites
 
-## Available Scripts
+In the project directory, you can run: `npm install` or `yarn install`
 
-In the project directory, you can run:
+## Running the tests
 
-### `npm start`
+Run `npm test` or `yarn test` in the console.
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+### Break down into end to end tests
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+Explain what these tests test and why
 
-### `npm test`
+## Deployment
 
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+This project was deployed with [Zeit](https://hackton-frontend-g3tpfw81r.now.sh/register).
 
-### `npm run build`
+## Built With
 
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
+* [React](https://reactjs.org/) - The web framework used.
+* [Redux](https://redux.js.org/) - State management tool.
+* [Styled-components](https://www.styled-components.com/) - Used to generate styles.
 
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+## Contributing
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Please read [CONTRIBUTING.md](https://gist.github.com) for details on our code of conduct, and the process for submitting pull requests to us.
 
-### `npm run eject`
+## Versioning
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+This is version 0.0.1 (Release canvas 1)
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Authors
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+* **Pascal Ulor** - *Team lead* - [github](https://github.com/PascalUlor)
+* **Mariam Adedeji** - *Team member* - [github](https://github.com/mariehposa)
+* **Mike Perry Attara** - *Team member* - [github](https://github.com/mikeattara)
+* **Justinas Vebra** - *Team member* - [github](https://github.com/vebradev)
+* **Mildred Otieno** - *Team member* - [github](https://github.com/)
```

Comment:
@mariehposa my github username is awuorm, it's missing xD

Reply:
I'm sorry about that, was a mistake. It has been updated by mike.